webservices/common/models/dates.py

```diff
@@ -30,6 +34,10 @@ def report_type_full(self):
         return clean_report_type(self.report.report_type_full)
 
 
+class ReportDateDisclosure(DisclosureMixin, ReportDate):
+    pass
```

Comment:
Okay, I see how you've accounted for the tests by doing this too.  Nice!

However, what do you think about it in the future if we generate an entirely new subset of data that _does_ account for all of the different schemas?  Should we think about this a little now or worry about it if that time comes?